identity/app/views/profile/emailPrefs.scala.html

```diff
@@ -33,58 +30,11 @@ <h2 class="form__heading">Your newsletters</h2>
                 The Guardian’s newsletters include content from our website, which may be funded by outside parties. Newsletters may also display information about Guardian News and Media's other products, services or events (such as Guardian Jobs or Masterclasses), chosen charities or online advertisements.
             </p>
         </div>
-        <div class="fieldset__fields">
-            @fragments.emailListCategories(emailPrefsForm,availableLists,emailSubscriptions)(request, messages)
+        <div class="fieldset__fields email-categories__categories">
```

Comment:
We are very loosely using BEM in these pages but since this is a "top block" there's no need to put `__categories`.

Also! very loose convention but I'd prefer this to be prefixed with `identity` to keep the identity-related CSS scoped, I'd personally go with something like `identity-email-categories-fieldset`